06-building-a-program-start-to-finish/solving-the-problem/output/inverted-index.js

```diff
@@ -77,7 +76,7 @@ class InvertedIndex {
       const fileIndexed = this.getIndex(fileName);
       const bookNames = fileIndexed.title;
       const result = InvertedIndexUtility
-      .filterIndexed(query, fileIndexed.index);
+      .filterIndexed(searchTerms, fileIndexed.index);
       searchResult = fileIndexed ?
```

Comment:
Please confirm this is the best way of using ternary operator. [See link.](https://github.com/airbnb/javascript#comparison--unneeded-ternary)